services/queue/schemas/v1/task-group-sealed.yml

```diff
@@ -0,0 +1,25 @@
+$schema: "/schemas/common/metaschema.json#"
+title:              "Task Group Sealed Message"
+description: |
+  Message written once a task group has been sealed.
```

Comment:
Is there a reason not to return the full task group response (e.g. including the sealed time?) (i.e. reuse the other json schema and not have a different one)

Reply:
Haven't thought about that! 
On one hand sealed timestamp cannot be changed, so one can query db/api to fetch the date if they are interested.
On the other hand it doesn't hurt to include more and to avoid clients do extra calls
:+1: